games/hedgewars/Portfile

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.0
+
+name                    hedgewars
+version                 0.9.22
+categories              games
+platforms               darwin
+license                 GPL-2
+maintainers             {gmail:ken.cunningham.webuse @kencu} openmaintainer
+
+description             Funny turn-based artillery game, featuring fighting Hedgehogs!
+long_description        ${description}
+homepage                https://www.hedgewars.org/
+
+master_sites            https://www.hedgewars.org/download/releases/
+use_bzip2               yes
+distname                ${name}-src-${version}
+
+checksums               rmd160  359fd026643e69446121b94ba301646812353c19 \
+                        sha256  b699c8971ff420c3edd6533527ae2e99040f1e79207c9140826945bcf0e62192
+
+cmake.out_of_source     yes
+
+depends_build-append    port:pkgconfig \
+                        port:ghc \
+                        port:hs-vector \
+                        port:hs-bytestring-show \
+                        port:hs-network \
+                        port:hs-dataenc \
+                        port:hs-hslogger \
+                        port:hs-utf8-string \
+                        port:hs-sha \
+                        port:hs-entropy \
+                        port:hs-zlib \
+                        port:hs-random
+                    
+depends_lib-append      path:bin/ffmpeg:ffmpeg \
+                        port:libogg \
+                        port:libsdl \
+                        port:libsdl_image \
+                        port:libsdl_ttf \
+                        port:libsdl_mixer \
+                        port:libsdl_net \
+                        port:zlib \
+                        port:libpng \
+                        port:glew \
+                        port:qt4-mac
+
+# don't use the sparkle selfupdate mechanism; use macports update process instead
+configure.args-append   -DNOAUTOUPDATE=1
+
+# don't add all of the macports support libraries into the app bundle
+configure.args-append   -DSKIPBUNDLE:BOOL=ON
+
+configure.args-append   -DQT_QMAKE_EXECUTABLE:FILEPATH=${prefix}/libexec/qt4/bin/qmake
```

Comment:
I guess there is always more to learn -- if I add the qmake portgroup, would that take care of this configure step for me? I can certainly give that a try!